assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -821,6 +825,34 @@ const baseSelectors = {
 
 		return screenWidgetContext || null;
 	} ),
+
+	/**
+	 * Gets the module's list of features.
+	 *
+	 * Returns a list of features of this module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(Array|undefined)} An array of features for the module; `undefined` if state is still loading.
+	 */
+	getModuleFeatures: createRegistrySelector( ( select ) => ( state, slug ) => {
+		const modules = select( STORE_NAME ).getModules();
+
+		// Return `undefined` if modules haven't been loaded yet.
+		if ( modules === undefined ) {
+			return undefined;
+		}
+
+		// A module with this slug couldn't be found; return `null` to signify the
+		// "not found" state.
+		if ( modules[ slug ] === undefined ) {
+			return null;
+		}
```

Comment:
I think we can remove this condition and simply return an empty array if there's no module for the slug as should already happen below. We shouldn't be using this selector to infer the existence of the module itself so we don't need another return value for that case 👍

Reply:
👍